addons/uom/models/uom_uom.py

```diff
@@ -75,6 +79,7 @@ def _unprotected_uom_xml_ids(self):
         default='reference', required=True)
     ratio = fields.Float('Combined Ratio', compute='_compute_ratio', inverse='_set_ratio', store=False)
     color = fields.Integer('Color', compute='_compute_color')
+    allow_label = fields.Boolean('Labelable', default=lambda self: self._default_labelable(), help="Print 1 product label per unit transferred")
```

Comment:
```suggestion
    allow_label = fields.Boolean('Labelable', default=lambda self: self._default_allow_label(), help="Print 1 product label per unit transferred")
```
if you rename the field, you have to rename the related function